PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/PlayerViewController.swift

```diff
@@ -100,55 +140,60 @@ class PlayerViewController: OOOoyalaPlayerViewController {
   }
   
   // Updates the time labels.
-  func updatePlayingInfoCenter() {
+  private func updatePlayingInfoCenter() {
     let playingInfoCenter = MPNowPlayingInfoCenter.default()
-    if var displayInfo = playingInfoCenter.nowPlayingInfo {
-      displayInfo[MPNowPlayingInfoPropertyPlaybackRate] = player.isPlaying() ? 1 : 0
-      displayInfo[MPMediaItemPropertyPlaybackDuration] = player.duration()
-      displayInfo[MPNowPlayingInfoPropertyElapsedPlaybackTime] = player.playheadTime()
-      playingInfoCenter.nowPlayingInfo = displayInfo
+    guard var displayInfo = playingInfoCenter.nowPlayingInfo, let player = ooyalaPlayerVC?.player else {
```

Comment:
Please, align `var displayInfo` and `let player` on 2 lines, i.e.
```
guard var displayInfo = ...,
      let player = ...
```

Reply:
Done